Cargo.toml

```diff
@@ -125,7 +125,9 @@ mime = "0.3"
 mockall = { version = "0.13.1", default-features = false }
 nix = { version = "0.29.0", default-features = false }
 once_cell = "1.7.2"
+openssl = { version = "0.10", default-features = false }
```

Comment:
reason why this is needed anymore?

Reply:
cdk and smdk uses cargo-generate that needs it vendored